value.go

```diff
@@ -306,32 +307,30 @@ func (s BNode) Native() interface{} { return s }
 // specific IRI type to their native equivalents.
 type StringConversion func(string) (Value, error)
 
+// Following the JSON-LD specification: https://w3c.github.io/json-ld-syntax/#conversion-of-native-data-types
 const (
-	nsXSD = `http://www.w3.org/2001/XMLSchema#`
-)
-
-// TODO(dennwc): make these configurable
-const (
-	defaultIntType   IRI = schema.Integer
-	defaultFloatType IRI = schema.Float
-	defaultBoolType  IRI = schema.Boolean
-	defaultTimeType  IRI = schema.DateTime
+	defaultIntType   IRI = xsd.Integer
+	defaultFloatType IRI = xsd.Double
+	defaultBoolType  IRI = xsd.Boolean
+	defaultTimeType  IRI = xsd.DateTime
 )
 
 func init() {
 	// int types
 	RegisterStringConversion(defaultIntType, stringToInt)
-	RegisterStringConversion(nsXSD+`integer`, stringToInt)
-	RegisterStringConversion(nsXSD+`long`, stringToInt)
+	RegisterStringConversion(xsd.Int, stringToInt)
+	RegisterStringConversion(xsd.Long, stringToInt)
```

Comment:
Forgot to register `schema:Integer`?

Reply:
yes

Reply:
It's registered with `RegisterStringConversion(defaultIntType, stringToInt)`

Reply:
But the `defaultIntType` is `xsd.Integer` now. We still need a registration for `schema.Integer`.

Reply:
Done.